percona/controllers/pgc/pgc.go

```diff
@@ -113,8 +113,26 @@ func (c *Controller) onAdd(obj interface{}) {
 
 	c.Queue.Add(key)
 	defer c.Queue.Done(key)
+
 	newCluster := obj.(*crv1.PerconaPGCluster)
+	nn := types.NamespacedName{
+		Name:      newCluster.Name,
+		Namespace: newCluster.Namespace,
+	}
+	if c.lockers.store == nil {
+		c.lockers = newLockStore()
+	}
 
+	l := c.lockers.LoadOrCreate(nn.String())
```

Comment:
@nonemax can we use more understandable names for vars?

Reply:
In this case I don't think it's needed because we create it few lines above from `NamespaceName` struct and use only here